### PR TITLE
Only check on csv is the file extension.

### PR DESCRIPTION
**Pull request by mannickutd**
Only check on csv is the file extension.

### CODE REVIEW

**Comment by quis** [comment]
Tested locally.

Made a little tweak: 
image

👍

File: app/main/validators.py [hunk 1]
```diff
@@ -16,12 +16,12 @@ def __call__(self, form, field):
...
 class CsvFileValidator(object):
 
-    def __init__(self, message='Not a csv file'):
+    def __init__(self, message='{} is not a CSV file'):
         self.message = message
 
     def __call__(self, form, field):
-        if not form.file.data.mimetype == 'text/csv':
-            raise ValidationError(self.message)
+        if field.data.filename.rsplit('.')[1].lower() != 'csv':
```
Comment (quis) [inline app/main/validators.py]: This gives an `IndexError` if you upload a file with no extension.

I think `field.data.filename.lower().endswith('.csv')` would work instead.

Comment (mannickutd) [inline app/main/validators.py]: very true

**Comment by quis** [comment]
👍

**Pull request merged**